code/BTCUtil.c

```diff
@@ -780,6 +780,17 @@ Datas ckdIndicesFromPath(const char *cStr)
     return DTPopDatas(result);
 }
 
+Data privKeyToHdWallet(Data privKey, const char *passphrase)
+{
```

Comment:
Add a `DataTrackPush();` on the first line. This "indents" the ARC scope by one -- it's not strictly needed but doing it allows the functions in this file (that require being inside an ARC scope) to be called by code not in any ARC scope.